### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by MrPerson**
Removes a bunch of unused or hardly used vars on /mob and /atom/movable.
Added defines for mob/living/var/mob_size so it's not using magic numbers.
Moves var/nightvision from /mob/living to /mob/living/carbon/alien since only aliens used it.

### CODE REVIEW

**Comment by Dorsisdwarf** [comment]
Spiders were supposed to use it, weren't they?

**Comment by Incoming5643** [comment]
nightvision was put in mob/living on my request because I eventually wanted to give it to some simple animals who logistically should have it (spiders/miningmobs/simplealiens) but I haven't gotten around to it.

Someone can always move it back when/if it's needed though

**Comment by MrPerson** [comment]
If I knew you actually planned on using it I wouldn't have moved it. But yeah, it's pretty easy to move around.

File: code/modules/mob/mob_defines.dm [hunk 1]
```diff
@@ -147,13 +144,9 @@
 
 	var/has_unlimited_silicon_privilege = 0 // Can they interact with station electronics
 
-	var/list/radar_blips = list() // list of screen objects, radar blips
-	var/radar_open = 0 	// nonzero is radar is open
```
Comment (Razharas) [inline code/modules/mob/mob_defines.dm]: Finally

Comment (Iamgoofball) [inline code/modules/mob/mob_defines.dm]: what was this for even

Comment (RemieRichards) [inline code/modules/mob/mob_defines.dm]: radar nanoaug, ancient.

**Pull request merged**